federation/pkg/federation-controller/service/dns.go

```diff
@@ -212,7 +212,9 @@ func (s *ServiceController) ensureDnsRrsets(dnsZoneName, dnsName string, endpoin
 			} else {
 				// Need to replace the existing one with a better one (or just remove it if we have no healthy endpoints).
 				// TODO: Ideally do these inside a transaction, or do an atomic update, but dnsprovider interface doesn't support that yet.
-				glog.V(4).Infof("Existing recordset %v is not equal to needed recordset %v, removing existing and adding needed.", rrset, newRrset)
+				glog.V(4).Infof("Existing recordset                %v", rrset)
```

Comment:
nit: Would recommend reconsolidating these into a single log statement, otherwise the individual log statements might get separated from each other due to multithreading.